beta-src/src/components/svgr-components/HomeIconSelected.js

```diff
@@ -0,0 +1,20 @@
+import * as React from "react";
+
+const SvgHomeIconSelected = ({iconState}) => (
+  <svg
+    width={40}
+    height={40}
+    // fill={iconState === "inactive" && "#000"}
```

Comment:
remove

Reply:
For some reason, removing these dimensions makes the svg default to 300px by 150px :(. So I kept them in the updated commit.